db/models/app_user.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../db');
+
+insert = (user) => (db('app_user').insert(user));
```

Comment:
Why all the parentheses?

Reply:
I did some reading, and some parentheses could be removed. I'll apply these changes.
(user) => (db('app_user').insert(user)) == user => db('app_user').insert(user);